backends-velox/src/test/scala/io/glutenproject/execution/VeloxFunctionsValidateSuite.scala

```diff
@@ -451,6 +451,13 @@ class VeloxFunctionsValidateSuite extends VeloxWholeStageTransformerSuite {
     }
   }
 
+  test("Test spark_partition_id function") {
+    runQueryAndCompare("""SELECT spark_partition_id(), l_orderkey
+                         | from lineitem limit 1""".stripMargin) {
```

Comment:
Let's use a larger value for `limit`, e.g., `limit 100`. We can also enlarge it for the above `monotonically_increasing_id` test.

Reply:
got! i will fix ahd upate for both functions in this PR later, thank you @PHILO-HE

Reply:
done